packages/edit-site/src/components/post-list/index.js

```diff
@@ -55,14 +71,21 @@ function useView( postType ) {
 			};
 		}
 		return defaultView;
-	}, [ isCustom, activeView, layout, postType, DEFAULT_VIEWS ] );
-	const [ view, setView ] = useState( selectedDefaultView );
-
+	}, [ isCustom, activeView, postType ] );
```

Comment:
The effect still depends on the layout I guess if we follow this we will need a lint ignore.